src/java/fr/paris/lutece/portal/web/user/AdminUserJspBean.java

```diff
@@ -845,7 +845,7 @@ public String getModifyAdminUser( HttpServletRequest request ) throws AccessDeni
         model.put( MARK_LOCALE, getLocale( ) );
         model.put( MARK_MAP_LIST_ATTRIBUTE_DEFAULT_VALUES, map );
         model.put( MARK_WORKGROUP_KEY_LIST, AdminWorkgroupService.getUserWorkgroups( getUser( ), getLocale( ) ) );
-        model.put( MARK_MINIMUM_PASSWORD_SIZE, AdminUserService.getIntegerSecurityParameter( AdminUserService.DSKEY_PASSWORD_MINIMUM_LENGTH ) );
```

Comment:
Because password modification has been moved to its own tab (see ```getModifyUserPassword```). This model variable was a leftover.